docs/native-code.md

```diff
@@ -0,0 +1,127 @@
+  - See also the guide for [moving from C++/CX to C++/WinRT](https://docs.microsoft.com/windows/uwp/cpp-and-winrt-apis/move-to-winrt-from-cx).
+  - Libraries built for .NET desktop framework cannot be directly accessed by UWP. You can create a .NET Standard library that calls into the .NET framework one, and call from the UWP app into the .NET Standard middleware.
+
+### Local testing and inner loop
+For internal development, you can deploy your app for test purposes by side-loading and deploying via [loose-file registration](https://docs.microsoft.com/windows/uwp/debug-test-perf/loose-file-registration). When building in **Debug** mode (which is the default), `npx react-native run-windows` performs loose-file registration of your app in order to install it locally. When running `npx react-native run-windows` with the `--release` switch, the CLI will install the real package onto your local machine. This requires the app to be signed and for the certificate it uses for signing to be trusted by the machine the app is going to be installed on. See [Create a certificate for package signing](https://docs.microsoft.com/windows/msix/package/create-certificate-package-signing) and [Intro to certificates](https://docs.microsoft.com/windows/uwp/security/certificates).
+
+
+## C# vs. C++ for Native Modules
+
+Although React Native for Windows supports writing modules in both C# and C++, you should be aware that your choice of language could impact performance of apps that consume your module. C# apps submitted to the store go through a process of cross-compilation to native code (the tools that allow this are the .NET Native toolchain), which improves the performance and startup times of these apps versus their non-"native-ized" counterparts. For more information about performance in .NET Native apps, see [Measuring startup improvement with .NET Native](https://docs.microsoft.com/dotnet/framework/net-native/measuring-startup-improvement-with-net-native).
+
+C# development brings engineering efficiencies to writing a module or an app. We strive to maintain parity in developer experiences between C# and C++. If your app or module already uses C#, you should feel empowered to continue to use C#. That said, modules that Microsoft contributes to will be written in C++ to ensure the highest level of performance and interoperability.
+
+<div class="warning">
+  <h3>Important</h3>
+  <h4>Mixing C# and C++</h4>
+</div>
+
+### App crashes when trying to load the C# component or instantiate one of its types
+**Error 0x80131040 “The located assembly’s manifest definition does not match the assembly reference”**

```

Comment:
I believe the 0.63+ C++ sample template does some of this by default now.

Reply:
The property exists, it is not on by default as it would bloat fully native apps unnecessarily.